code/_onclick/ai.dm

```diff
@@ -45,7 +45,7 @@
 			return //So there's no point messaging admins
 		message_admins("[ADMIN_LOOKUPFLW(src)] was kicked because they failed can_see on AI click of [A] (Turf Loc: [ADMIN_VERBOSEJMP(pixel_turf)]))")
 		log_admin("[key_name(src)] was kicked because they failed can_see on AI click of [A] (Turf Loc: [AREACOORD(pixel_turf)])")
-		to_chat(src, "<span class='reallybig'>You have been automatically kicked because you clicked a turf you shouldn't of been able to see as an AI. You should reconnect automatically. If you do not, you can reconnect using the File --> Reconnect button.</span>")
+		to_chat(src, "<span class='reallybig'>You have been automatically kicked because you clicked a turf you shouldn't have been able to see as an AI. You should reconnect automatically. If you do not, you can reconnect using the File --> Reconnect button.</span>")
```

Comment:
```suggestion
		to_chat(src, "<span class='reallybig'>You have been kicked because you clicked a turf you shouldn't have been able to see as an AI. You will reconnect automatically. If you do not, you can reconnect using the File --> Reconnect button.</span>")
```

Reply:
no